File: ML/Coderaman-tech_AmanKumarShrivastava_2024cse1155_2/GettingStarted/Is_a_number_Prime.cpp
#include <iostream>
using namespace std;
int main(){
    int t;
    cin >> t;

    while(t--){
        int n;
        cin>>n;
        int count=0;
        for(int i=2;i<n;i++){
            if(n%i==0){
                cout<<"not prime"<<"\n";
                count=1;
                break;
            }
        }
        if(count==0)
        cout<<"prime"<<"\n";
    }
    return 0;
}

File: Web/AyushK-101_AyushKumar_2024cse1170_2/GettingStarted/IsANumberPrime.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
      int t=scn.nextInt();
      
      for(int i=1;i<=t;i++)
      {
          int n=scn.nextInt();
          
          int count=0;
          
          for(int j=2;j*j<=n;j++)
          {
              if(n%j==0)
              {
                  count++;
                  break;
              }
          }
          if(count==0)
          {
              System.out.println("prime");
          }
          else
          {
              System.out.println("not prime");
          }
          
      }
   }